### ISSUE TITLE
netsfs.f: "0.003: Table distribution type without prob table around line 45"

### ISSUE THREAD

**Issue by yangx-jy** [filebench/filebench#133]
Hi,

Running netsfs.f triggers the following stderr:
-----------------------------------------------------------------
0.003: Table distribution type without prob table around line 45
0.003: Table distribution type without prob table around line 53
0.003: Table distribution type without prob table around line 66
-----------------------------------------------------------------

It seems that all workloads including "...randvar(type=tabular..." have the same issue.

Best Regards,
Xiao Yang

**Comment by sectorsize512** [comment]
Yeah, the syntax for randvars was changed. We need to address this.
